src/com/elikill58/negativity/universal/Cheat.java

```diff
@@ -185,12 +185,12 @@ public String getKey() {
 			return placeholders;
 		}
 		
-		public String compileDefault(Cheat c) {
-			return key == "" ? key : "\n" + TranslatedMessages.getStringFromLang(TranslatedMessages.DEFAULT_LANG, "hover." + c.getKey() + "." + getKey(), getPlaceholders());
+		public String compile(Cheat c) {
+			return key == null ? "" : "\n" + TranslatedMessages.getStringFromLang(TranslatedMessages.DEFAULT_LANG, "hover." + c.getKey() + "." + getKey(), getPlaceholders());
 		}
 		
 		public String compile(Cheat c, NegativityPlayer np) {
-			return key == "" ? key : "\n" + TranslatedMessages.getStringFromLang(np.getAccount().getLang(), "hover." + c.getKey() + "." + getKey(), getPlaceholders());
+			return key == null ? "" : "\n" + TranslatedMessages.getStringFromLang(np.getAccount().getLang(), "hover." + c.getKey() + "." + getKey(), getPlaceholders());
```

Comment:
```suggestion
			return key == null ? key : "\n" + TranslatedMessages.getStringFromLang(np.getAccount().getLang(), "hover." + c.getKey() + "." + getKey(), getPlaceholders());
```
Il faut quand même retourner la clé (pareil pour l'overload)

Reply:
la clé est null, donc a quoi bon la retourner ?